src/com/company/IntListImpl.java

```diff
@@ -0,0 +1,220 @@
+        Integer[] arr = array;
+        sort(arr);
+        Integer element = searchBin(array, item);
+        if (element != 0) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    @Override
+    public Integer indexOf(Integer item) {
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == item) {
+                return i;
+            }
+        }
+        return -1;
+    }
+
+    @Override
+    public Integer lastIndexOf(Integer item) {
+        for (int i = array.length; i >= 0; i--) {
+            if (array[i - 1] == item) {
+                return i - 1;
+            }
+        }
+        return -1;
+    }
+
+    @Override
+    public Integer get(int index){
+        if (array[index] == 0) {
+            throw new NullElementInTheArray("В ячейке под индексом " + index + " объект отсутствует");
+        } else {
+            return array[index];
+        }
+    }
+
+    @Override
+    public boolean equals(IntList otherList) {
+        boolean result = true;
+        if (otherList == null) {
+            throw new NullElementInTheArray("Один из объектов отсутствует!");
+        } else if (array.length != otherList.size()) {
+            return false;
+        } else {
+            int eL = 0;
+            for (int i = 0; i < array.length; i++) {
+                if (array[i] == otherList.get(i)) {
+                    eL++;
+                }
+            }
+            if (eL == size) {
+                return result;
+            }
+        }
+        return result;
+    }
+
+    @Override
+    public int size() {
+        return size;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        return size() == 0;
+    }
+
+    @Override
+    public void clear() {
+        for (int i = 0; i < array.length; i++) {
+            array[i] = 0;
+        }
+        size = 0;
+    }
+
+    @Override
+    public Integer[] toArray() {
+        Integer[] array = new Integer[10];
+        return array;
+    }
+
+    private void sort(Integer[] arr) {
+        for (int i = arr.length - 1; i > 0; i--) {
+            int maxEl = Integer.MIN_VALUE;
+            int index = 0;
+            for (int j = 0; j <= i; j++) {
+                if (arr[j] > maxEl) {
+                    maxEl = arr[j];
+                    index = j;
+                }
+            }
+            int a = arr[i];
+            arr[i] = maxEl;
+            arr[index] = a;
+        }
+    }
+
+    private Integer searchBin(Integer[] array, int element) {
+        int min = 0;
+        int max = array.length - 1;
+        while (min <= max) {
+            int mid = (min + max) / 2;
+            if (element == array[mid]) {
+                return array[mid];
+            }
+            if (element < array[mid]) {
+                max = mid - 1;
+            } else {
+                min = mid + 1;
+            }
+        }
+        return Integer.parseInt("Данный элемент, не найден");
```

Comment:
как это должно работать? parseInt может превратить строку в число, если строка содержит только число, например
```suggestion
        return Integer.parseInt("12");
```